apps/oxlint/src/lint/mod.rs

```diff
@@ -116,6 +116,14 @@ impl Runner for LintRunner {
         };
 
         enable_plugins.apply_overrides(&mut oxlintrc.plugins);
+
+        // TODO: add support for printing the configuration for a specific path when overrides supported.
+        let oxlintrc_for_print = if let Some(PrintConfig::All) = &misc_options.print_config {
```

Comment:
> even advertise it as a CLI option

I wanted `--print-config`/`--print-config all` to represent 'all', but I didn't find a proper way in bpaf, to avoid breaking change, maybe warn for now?